apps/frontend/static_src/js/feedback/index.js

```diff
@@ -43,15 +46,15 @@ export const handleFeedback = () => {
 
     let feedbackPk = null;
 
-    const post_feedback = async (feedback) => {
+    const postFeedback = async (feedback) => {
         try {
             const res = await fetch(window.location.pathname, {
                 method: 'POST',
                 body: JSON.stringify({
                     feedback,
                 }),
                 headers: {
-                    'X-CSRFToken': csrftoken,
+                    'X-CSRFToken': getCookie('csrftoken'),
```

Comment:
I’m getting a CSRF failure on those forms but I don’t think this is related to those changes, will open separately.

Reply:
Hmm. I did test this a lot locally - hopefully I didn't break it.